<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Image">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>141</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdauga_Concurent"/>
    <addaction name="actionAdauga_Categorie"/>
    <addaction name="actionAdauga_Club"/>
    <addaction name="actionStergeConcurent"/>
    <addaction name="actionSterge_Categorie"/>
    <addaction name="actionSterge_Club_Sportiv"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdauga_Concurent">
   <property name="text">
    <string>Adauga Concurent</string>
   </property>
  </action>
  <action name="actionAdauga_Categorie">
   <property name="text">
    <string>Adauga Categorie</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAdauga_Club">
   <property name="text">
    <string>Adauga Club Sportiv</string>
   </property>
  </action>
  <action name="actionStergeConcurent">
   <property name="text">
    <string>Sterge Concurent</string>
   </property>
  </action>
  <action name="actionSterge_Categorie">
   <property name="text">
    <string>Sterge Categorie</string>
   </property>
  </action>
  <action name="actionSterge_Club_Sportiv">
   <property name="text">
    <string>Sterge Club Sportiv</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
